src/OpenLoco/Entities/Entity.h

```diff
@@ -24,7 +24,8 @@ namespace OpenLoco
     enum class EntityBaseType : uint8_t
     {
         vehicle = 0,
-        misc
+        misc,
+        null = 0xFF
```

Comment:
I'm not sure about this last entry. Adding `EntityBaseType::null` will introduce the need to handle the `null` case in all switch statements handling an `EntityBaseType`.

Reply:
There won't be many of them and I think this is the most logical. This is already being used in the load branch as well so would mean changing that.